### DISCUSSION TITLE
How to create dependencies between routes

### DISCUSSION THREAD

**Discussion by FreddyHunter** [google/or-tools#3007] [Routing (and legacy CP) questions]
I would like to know if there is a way to create dependencies between routes. I mean, imagine that I have truck A that carry product 1 for a specific location.  There is also truck B which carry product 2 for the same specific location. And finally a truck C, which can just start to distribute product 1 and product 2 after truck A and truck B have unloaded at that specific location. Any ideas?

**Comment by Mizux** [comment]
First you need to duplicate the node
N_1 N_2 and N_C
Then add constraint time(n_1) <= time(n_c) and time(n_2) <= time(n_c)
